src/lightning_app/utilities/secrets.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict, List
+
+from lightning_app.utilities.cloud import _get_project
+from lightning_app.utilities.network import LightningClient
+
+
+def _names_to_ids(secret_names: List[str]) -> Dict[str, str]:
+    """Returns the name/ID pair for each given Secret name.
+
+    Raises a `ValueError` if any of the given Secret names do not exist.
+    """
+    lightning_client = LightningClient()
+
+    project = _get_project(lightning_client)
+    secrets = lightning_client.secret_service_list_secrets(project.project_id)
```

Comment:
We have another PR to make all the client call arguments key/value pair instead of positional. Adding here too, so we won't miss it

```suggestion
    secrets = lightning_client.secret_service_list_secrets(project_id=project.project_id)
```